yarn.lock

```diff
@@ -1231,9 +1231,9 @@
     paraswap "^5.2.0"
     paraswap-core "^1.0.2"
 
-"@cowprotocol/ethflowcontract@cowprotocol/ethflowcontract.git#0.0.0-rc4-artifacts":
+"@cowprotocol/ethflowcontract@cowprotocol/ethflowcontract.git#v1.0.0-artifacts":
   version "0.0.0-rc.4"
```

Comment:
Aha! Found out why this version didn't update.
Because the project's package.json version remained the same https://github.com/cowprotocol/ethflowcontract/blob/a751d53bf248881bbee27df42ddf73ffc69f7c0a/package.json#L14

cc @anxolin @fedgiac